### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by rtonata88**
In the PR I have implemented all the features for the App with the following screens:

- Splash screen: A simple page with the name of the app.
- Sign up and log in pages
- Home page (catagories page)
- Expenses page
- "Add a new category" page
- "Add a new expense" page

### CODE REVIEW

File: app/views/expenses/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<div id="expenses">
+  <% @expenses.each do |expense| %>
+    <%= render expense %>
+    <p>
+      <%= link_to "Show this expense", expense %>
+    </p>
+  <% end %>
+</div>
+
+<%= link_to "New expense", new_expense_path %>
```
Comment (DeliceLydia) [inline app/views/expenses/index.html.erb:14]: - [ ] All buttons should have a color like in the design `#5fb523` and please follow the given [design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)

Comment (DeliceLydia) [reply app/views/expenses/index.html.erb:14]: - [ ] Design of your application should follow the given one:
- Colors
- Typography: font face, size and weight.
- Layout: composition and space between elements.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,71 @@
...
+### Demo
+[Video Demo](https://www.loom.com/share/8c12fa3bafc34cc28052cafccaa495a3)
+
+## Authors
+
+👤 **Richard Chambula**
+
+- GitHub: [@rtonata88](https://github.com/rtonata88)
+- Twitter: [@rtonata](https://twitter.com/rtonata)
+- LinkedIn: [Richard Chambula](https://www.linkedin.com/in/richard-chambula-49198425/)
```
Comment (DeliceLydia) [inline README.md]: - [x] Please deploy your applications and add a live demo in your README

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @rtonata88, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- App is working as expected
- Awesome README
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### You Rock!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md [hunk 3]
```diff
@@ -1,24 +1,72 @@
...
+
+Feel free to check the [issues page](https://github.com/oliverscz/blog-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rustam-tolipov) [inline README.md:72]: - [ ] [OPTIONAL] You have added the MIT license link in the readme but there is no MIT.md file itself. So please add the license file also 👍

File: README.md [hunk 4]
```diff
@@ -1,24 +1,72 @@
-# README
+# Budgeta
```
Comment (rustam-tolipov) [inline README.md:2]: - [ ] [OPTIONAL] Let's also add the screenshot of your app also 📷 to the readme file 📃.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @rtonata88 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**